packages/sw-precaching/src/lib/controllers/base-cache-manager.js

```diff
@@ -32,6 +32,28 @@ class BaseCacheManager {
   }
 
   /**
+   *This method gives access to the cache used by thie caching manager.
+   * @return {String} The cache name used for this manager.
+   */
+  getCacheName() {
+    return this._cacheName;
+  }
+
+  /**
+   * This method returns an array of URL's that will be cached by this
+   * cache manager.
+   * @return {Array<String>} An array of URLs that will be cached.
+   */
+  getCachedUrls() {
+    const urls = [];
```

Comment:
Please ignore if you find this less readable, but you could rewrite as:

```
return [...this._entriesToCache.keys()]
  .map(url => new URL(url, location).href)
```

or equivalently:

```
return Array.from(this._entriesToCache.keys())
  .map(url => new URL(url, location).href)
```